homeworks/week02/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Week 2, Homework</title>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script src="js/main.js"></script>
+
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+
+    <script>
+function render(movieArray, displayWide) {
```

Comment:
@nmadd, @ckuang, are we cool with scripts inside the HTML file? Or break this out into a separate .js file?